etc/deploy.sh

```diff
@@ -297,7 +299,8 @@ install_charts() {
     --set swan.jupyterhub.hub.config.KeyCloakAuthenticator.oidc_issuer=https://${HOSTNAME} \
     --set swan.jupyterhub.hub.config.KeyCloakAuthenticator.oauth_callback_url=https://${HOSTNAME}/swan/hub/oauth_callback \
     --set swan.jupyterhub.ingress.hosts="{${HOSTNAME}}" \
-     sciencebox sciencebox/sciencebox
+    --set nginx.ingress.hostname=${HOSTNAME} \
+     sciencebox ../charts/sciencebox
```

Comment:
This was for testing locally. Should be updated to `sciencebox sciencebox/sciencebox`

Reply:
Yep, problem is it needs to be tagged. Let's test proper locally first

Reply:
I am marking as draft just to avoid merging before we revert the use of local charts.